### ISSUE TITLE
Insufficient background color contrast between header and banner

### ISSUE THREAD

**Issue by ladyvalentine** [smashstack-randi/qa-lady#8]
**OS**: Windows
**Browser**: Google Chrome / Mozilla FF
**Browser Version**: Latest
**Link to the page**: http://50.28.86.18/qa-exam/dist/test.html
**Section(s) where the issue is present**: Header and Banner

**Steps To Reproduce**:
1. Access **Smashstack** - _http://50.28.86.18/qa-exam/dist/test.html_

**Actual**: Insufficient background color contrast between header and banner (*Please refer to the 2 shade of green for header and banner*)

![image](https://cloud.githubusercontent.com/assets/22239457/18599720/5de68c48-7c8c-11e6-946f-7e9f3669a13f.png)

**Expected**: Sufficient background color contrast between header and banner for proper (section/area) distinction
